File: src/_1ProgrammingBasicsWithJavaMay2023/_8Exam/_1MainExam/_02FootballKit.java
package _1ProgrammingBasicsWithJavaMay2023._8Exam._1MainExam;

import java.util.Scanner;

public class _02FootballKit {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double priceOfTShirt = Double.parseDouble(scanner.nextLine());
        double sumNeededForBall = Double.parseDouble(scanner.nextLine());
        double priceOfShorts = priceOfTShirt * 0.75;
        double priceOfSocks = priceOfShorts * 0.20;
        double priceOfButtons = 2 * (priceOfTShirt + priceOfShorts);
        double totalSum = priceOfTShirt + priceOfShorts + priceOfSocks + priceOfButtons;
        double sumAfterDiscount = totalSum * 0.85;
        if (sumAfterDiscount >= sumNeededForBall) {
            System.out.printf("Yes, he will earn the world-cup replica ball!%n");
            System.out.printf("His sum is %.2f lv.", sumAfterDiscount);
        } else {
            double diff = sumNeededForBall - sumAfterDiscount;
            System.out.printf("No, he will not earn the world-cup replica ball.%n");
            System.out.printf("He needs %.2f lv. more.", diff);
        }
    }
}

File: src/_1ProgrammingBasicsWithJavaScriptNovember2024/_8Exam/_1MainExam/_02FootballKit.js
function footballKit(priceOfTShirt, sumNeededForBall) {
    const priceOfShorts = priceOfTShirt * 0.75;
    const priceOfSocks = priceOfShorts * 0.20;
    const priceOfButtons = 2 * (priceOfTShirt + priceOfShorts);
    const totalSum = priceOfTShirt + priceOfShorts + priceOfSocks + priceOfButtons;
    const sumAfterDiscount = totalSum * 0.85;
    if (sumAfterDiscount >= sumNeededForBall) {
        console.log(`Yes, he will earn the world-cup replica ball!`);
        console.log(`His sum is ${sumAfterDiscount.toFixed(2)} lv.`);
    } else {
        const diff = sumNeededForBall - sumAfterDiscount;
        console.log(`No, he will not earn the world-cup replica ball.`);
        console.log(`He needs ${diff.toFixed(2)} lv. more.`);
    }
}

footballKit(25,
100
)

footballKit(55,
310
)

footballKit(59.99,
500
)